data/mods/Xedra_Evolved/furniture_and_terrain/furniture.json

```
@@ -206,7 +206,7 @@
     "id": "f_dreamforged_bed_of_restful_repose",
     "name": "dreamforged bed of restful repose",
```

```suggestion
    "name": { "str": "dreamforged bed of restful repose", "str_pl": "several dreamforged beds of restful repose" },
```
So we can avoid more pluralization fixing PRs in the future

---

`1 dreamforged bed of restful repose`,
`4 several dreamforged beds of restful repose`?

---

Yeah I'm not sure why "several" is there, but I copied this from the proposed plural form of the folded version

---

> Furnitures don't have plurals though. The item name is already fixed by #71104 too.

Hi Qrox, 
What do I need to do to make sure basic build is running on XE PRs correctly? Or dud it just fail weirdly this time?

---

I think the basic build failed in this PR due to `data/mods/Xedra_Evolved/items/dreamsmith.json` not being formatted. XE PRs seem fine, because #71104 passed all its checks.